static/docs/get-started/install.md

```diff
@@ -1,8 +1,8 @@
 # Install
 
 There are a few recommended ways to install DVC: OS-specific package/installer,
-`pip`, `conda` (Anaconda), and Homebrew. (Depending on your OS, some of these may
-be not available.)
+`pip`, `conda` (Anaconda), and Homebrew. (Depending on your OS, some of these
```

Comment:
This one was added automatically by the formatter. Let's leave it here, if it is okay with you guys :)